tests/test_reclaim_policy.py

```diff
@@ -0,0 +1,271 @@
+
+    """
+    log.info(f'creating a new project {project_name}')
+    assert NAMESPACE.create(resource_name=project_name, out_yaml_format=False), \
+        f"created project {project_name}"
+    return True
+
+
+def delete_namespace(project_name):
+    """
+    Deletes the project
+    Args:
+        project_name (str): Project to be deleted
+
+    Returns:
+        (bool): True if deletion is successful, asserts otherwise
+
+    """
+    log.info(f"deleting project {project_name}")
+    assert NAMESPACE.delete(resource_name=project_name), \
+        f"Deleted project {project_name}"
+    return True
+
+
+def create_storage_class(**kwargs):
+    """
+    Creates a storage class
+    Args:
+        **kwargs: Takes in multiple values for storage class params
+
+    Returns:
+        (bool): True if creation is successful, False otherwise
+
+    """
+
+    file_sc = templating.generate_yaml_from_jinja2_template_with_data(
+        SC_YAML, **kwargs
+    )
+    with open(TEMP_SC_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_sc, yaml_file, default_flow_style=False)
+    return SC.create(yaml_file=TEMP_SC_YAML_FILE)
+
+
+def delete_storage_class(sc_name):
+    """
+    Deletes a storage class
+    Args:
+        sc_name(str): Storage class to be deleted
+
+    Returns:
+        (bool): True if deletion is successful, False otherwise
+
+    """
+
+    log.info(f'deleting storage class: {sc_name}')
+    return SC.delete(resource_name=sc_name)
+
+
+def create_pvc(**kwargs):
+    """
+    Creates a Persistent Volume Claim
+    Args:
+        **kwargs:
+
+    Returns:
+        (bool): True if creation is successful, False otherwise
+
+    """
+
+    file_pvc = templating.generate_yaml_from_jinja2_template_with_data(
+        PVC_YAML, **kwargs
+    )
+    with open(TEMP_PVC_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_pvc, yaml_file, default_flow_style=False)
+        log.info(f"Creating new Persistent Volume Claim")
+    assert PVC.create(yaml_file=TEMP_PVC_YAML_FILE)
+    return PVC.wait_for_resource(
+        resource_name=kwargs['pvc_name'], condition='Bound'
+    )
+
+
+def delete_pvc(pvc_name):
+    """
+    Deletes a PVC and its underlying PV.
+     Args:
+        pvc_name: Name of the PVC to be deleted
+
+    Returns:
+        (bool): True if deletion is successful, asserts otherwise
+
+    """
+
+    log.info(f"Deleting the Persistent Volume Claim {pvc_name}")
+    assert PVC.delete(resource_name=pvc_name), "Deletion of PVC failed"
+    return True
+
+
+def list_ceph_images(pool_name='rbd'):
+    log.info("checking if the rbd image is deleted from ceph cluster")
+    return ocp.exec_ceph_cmd(f"rbd ls {pool_name} --format=json")
+
+
+def rbd_rm_volume(vol, pool_name='rbd'):
+    """
+
+    Args:
+        vol(str): rbd image name, which is also PV name
+        pool_name(str): ceph pool used. Defaults to rbd
+
+    Returns:
+        None if the deletion is successful or
```

Comment:
In that case, it would be better to raise an exception (assertionError for example) in case the returned value from exe_cmd_on_pod is a command failure one.
And in case of success, the function should be void